### PR TITLE
Expose LND RPC/REST for lndconnect

### LINKED ISSUE #236: Add lndconnect
Add lndconnect to allow users to connect their Umbrel's lightning node to wallets such as Zap and Zeus.

### PR DESCRIPTION
**Pull request by mayankchhabra**
Resolves #236.

This PR exposes LND's gRPC and REST over Tor so `umbrel-manager` could use it to encode `lndconnect` URL schemes.

Testing:

**Zap desktop**
[I couldn't get Zap on desktop to work](https://github.com/LN-Zap/zap-desktop/issues/3620#issuecomment-693972786) with gRPC over Tor. It does work on the local network though (by selecting `lndconnect gRPC` from the dropdown).

**Zap iOS**
Doesn't scan the QR for some reason but worked without an issue when copy-pasting the generated URL. 

**Zeus Wallet**
It was able to scan the QR code and connect locally. [It doesn't support Tor yet](https://github.com/ZeusLN/zeus/blob/master/docs/Bounties.md#tor-on-ios).

---
Before merge:
- [x] Revert to official docker images 

Related: 
- https://github.com/getumbrel/umbrel-manager/pull/46
- https://github.com/getumbrel/umbrel-dashboard/pull/218

### CODE REVIEW

File: templates/lnd-sample.conf [hunk 1]
```diff
@@ -6,6 +6,9 @@ maxpendingchannels=3
 minchansize=10000
 accept-keysend=true
 tlsextraip=10.11.1.2
+tlsextraip=192.168.0.0/16
+tlsextradomain=umbrel.local
+tlsextradomain=umbrel-testnet.local
```
[the remark this answers is not in the document]
Comment (mayankchhabra) [reply templates/lnd-sample.conf:11]: Yeah, that makes sense!

File: templates/lnd-sample.conf [hunk 2]
```diff
@@ -6,6 +6,9 @@ maxpendingchannels=3
 minchansize=10000
 accept-keysend=true
 tlsextraip=10.11.1.2
+tlsextraip=192.168.0.0/16
```
[the remark this answers is not in the document]
Comment (mayankchhabra) [reply templates/lnd-sample.conf:9]: It's to make sure the TLS cert is also valid for local IP range. Eg. the user is using an android phone and wants to connect to Zeus via lndconnect (which doesn't support Tor), then umbrel.local wouldn't work for them in the lndconnect URL, but they can then replace umbrel.local with the device IP to make it work.

[the remark this answers is not in the document]
Comment (lukechilds) [reply templates/lnd-sample.conf:9]: I'm not sure we should do that, I can't think of anything off the top of my head but it doesn't seem very safe to use a certificate that's valid for every IP on the local network.

I personally think we should just not allow connecting over the local network at all.

It kind of makes sense for a desktop wallet but it's pretty confusing for a mobile wallet to only be able to connect when you're at home and connected to WiFi.

[the remark this answers is not in the document]
Comment (lukechilds) [reply templates/lnd-sample.conf:9]: Also does this definitely work?

This stack exchange answer suggests it's not possible to add an IP range to a cert:

https://security.stackexchange.com/questions/91368/ip-range-in-ssl-subject-alternative-name

Did you get errors connecting via IP without this?

**Review by lukechilds — APPROVED** [review]
Looks great, don't forget to revert image names!

**Comment by mayankchhabra** [comment]
Almost forgot it, almost... haha

**Review by lukechilds — DISMISSED** [review]
I don't have access to an iOS device but I've tested on Zap/Zeus on Android and it works great!

**Comment by huumn** [comment]
AFAICT The reason why lightning grpc doesn't work over Tor is that grpc in most clients forces use of ssl when providing additional credentials (macaroon). 

See: https://github.com/alexbosworth/ln-service/issues/96. 

Thus, clients that use grpc will not be able to connect over Tor to umbrel's grpc hidden service endpoints unless a reverse proxy is put in front.

**Pull request merged**